requirements/generated/requirements-dev.txt

```diff
@@ -66,31 +70,50 @@ markupsafe==2.1.5
 nodeenv==1.8.0
     # via pre-commit
 packaging==24.0
-    # via build
+    # via
+    #   build
+    #   pytest
 pip-tools==7.4.1
     # via -r requirements/source/requirements-dev.in
 platformdirs==4.2.2
     # via virtualenv
+playwright==1.43.0
```

Comment:
this is 1.43, but circle is using 1.44. It would seem appropriate to use the same version somehow?

Reply:
This is a good catch. `1.44` is the newest but when using `pytest-playwright` which is used to install all playwright dependencies, it keeps install `1.43`! Turns out you'll need to add playwright itself to the requirements files with it's own version control.